bananalyzer/data/generate_examples.py

```diff
@@ -101,6 +120,67 @@ async def llm_annotate_example(
     return details
 
 
+async def download_as_har(url: str) -> str:
+    async with async_playwright() as playwright:
+        browser = await playwright.chromium.launch(headless=True)
+        context = await browser.new_context()
+        page = await context.new_page()
+
+        # Generate random example ID and create new directory
+        example_id = str(uuid.uuid4())
+        example_dir_path = f"./static/{example_id}"
+        os.makedirs(example_dir_path, exist_ok=True)
+
+        # Make browser record all network responses into a HAR file
+        now = datetime.now()
+        timestamped_har_path = os.path.abspath(f"./static/{example_id}/cache/bananas." + now.strftime("%s") + ".har")
```

Comment:
typically i perfer pathlib.Path but this works too

Reply:
I'll switch it to be consistent with the rest of the code, thank you for the tip Adam!